schemas/alternate-registration-schema.json

```diff
@@ -0,0 +1,93 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "A representation of multiple registrations for the same entity",
+  "type": "object",
+  "properties": {
+    "data_type": {
+      "enum": [
+        "alternate-registration"
+      ]
+    },
+    "alternate_entities": {
+      "description": "TBC",
```

Comment:
"array of entities that are linked via alternate registrations"

Reply:
✅